docs/source/gbd2019_models/causes/postpartum_depression/index.rst

```diff
@@ -0,0 +1,191 @@
+
+PPD refers to non-psychotic depressive episodes in the postpartum period that persist for more than two weeks and shares the same diagnostic criteria as major depressive disorder in the DSM-5 [Shorey-et-al-2018]_. Specifically, major depressive disorders are marked by depressed mood or loss of interest/pleasure that represent a change from the person's baseline and impaired functioning observed across social, occupational, and educational domains. Additional symptoms may include excessive sleeping or insomnia; change in eating, appetite, or weight; agitated or slow motor activity; fatigue; feeling worthless or inappropriately guilty; trouble concentrating; and repeated thoughts about death. [GBD-2019-Capstone-Appendix-PPD]_
+
+GBD 2019 Modeling Strategy
+--------------------------
+
+Postpartum depression is not specifically estimated by the GBD. Rather, we will inform the prevalence of postpartum depression from a recent review and meta-analysis performed by [Shorey-et-al-2018]_. This review focused on healthy mothers with no prior history of postpartum depression, but found similar results to previous evaluations of postpartum depression prevalence among the broader population. [Shorey-et-al-2018]_ reported that the incidence of postpartum depression was equal to 12% (95% CI 0.04–0.20).
+
+However, the GBD estimated that the average duration of major depressive disorder was 0.65 (95% UI: 0.59, 0.70) of one year (as informed through analysis of longitudinal studies on remission from major depressive disorder and the assumption that 40 years is the maximum duration of the condition), which we will use to inform the duration of postpartum depression in our simulation [GBD-2019-Capstone-Appendix-PPD]_ (page 1013). 
+
+Additionally, the GBD models severity distributions and associated disability weights for major depressive disorder, summarized in the table below [GBD-2019-Capstone-Appendix-PPD]_ (page 1013). Notably, the GBD defines major depressive disorder (MDD) as an episodic mood disorder involving the experience of one or more major depressive episode(s). Notably, the percent of case severity was determined from the US National Epidemiological Survey on Alchol and Related Conditions (conducted in two waves from 2001-2002 and 2004-2005) and the Australian National Survey of Mental Health and Wellbeing of Adults. Notably, burden due to major depressive disorder in GBD begins in the 1 to 4 year age group.
+
+.. list-table:: Major depressive disorder disability weights
+  :header-rows: 1
+
+  * - Severity
+    - Percent of cases
+    - Disability weight
+  * - Asymptomatic
```

Comment:
Is GBD's construct of Asymptomatic MDD relevant to PPD?  Maybe we should redistribute this 13% to states with non-zero disability weight.

Reply:
Hmm, PPD shares the same diagnostic criteria as MDD in the DSM, so I haven't come across information that would suggest a different diagnostic scheme. I think that "asymptomatic" in GBD might refer to asymptomatic in terms of YLDs (DW=0), but not asymptomatic in that they are not displaying symptoms of MDD. 

The description of mild MDD is as follows: "Feels persistent sadness and has lost interest in usual activities. The person sometimes sleeps badly, feels tired, or has trouble concentrating but still manages to function in daily life with extra effort." I imagine that the asymptomatic sequelae could be something similar, but specifies "with little to no extra effort" instead. I know the diagnostic instruments first assess presence/frequency of symptoms and then assess severity in terms of the degree to which it interferes with daily functioning, so I think the latter portion could be part of how they distinguish asymptomatic and mild cases.

This is not clear in the methods appendix, though, so I'll see if I can get some more info from the GBD modeler to make sure!